src/Forms/Prism.Forms/Navigation/IInitialize.cs

```diff
@@ -1,7 +1,15 @@
 ﻿namespace Prism.Navigation
 {
+    /// <summary>
+    /// Synchronous initialization of navigation
```

Comment:
No need to add "synchronous" in the description. This class is used to initialize views and viewmodels during the navigation process

Reply:
done